docs/debezium-aws-rds-config-scripts

```diff
@@ -0,0 +1,361 @@
+= A Step-by-Step Guide on Streaming Data from AWS RDS Multi-AZ PostgreSQL DB to a Standlone PostgreSQL DB Using Debezium 0.7.1 Connector
```

Comment:
The title is a bit long.

Reply:
Hi @gunnarmorling  @jpechane , Sorry for the delay. I'm so busy with my job, still I wanna do my best to contribute the project. I read the instructions on the github, it was a little bit time-consuming.  Actually, I added all my images, scripts etc on github at the link https://github.com/fatihmehmetkose/debezium-aws-rds-config-scripts.

To make it faster,I need your help. Let me change the doc/blog content as your recommendation. For deploying the blog, I thought, it will be much faster if you deploy it.